<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Author</class>
 <widget class="QDialog" name="Author">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>636</width>
    <height>444</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="nickname">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>欢迎我们的作者沙雕陆昊（改昵称</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>我的粉丝</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>我的书籍</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>收藏此书的用户</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>我的信息</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="find">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>80</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>查找</string>
   </property>
  </widget>
  <widget class="QComboBox" name="addContext">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>360</y>
     <width>87</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>书籍</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>章节</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="add">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>360</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fixProfile">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>250</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>修改简介</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>130</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="findText">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>80</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>360</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="profile">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>140</y>
     <width>161</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="exit">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>360</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>注销</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
